docs/operations/index.rst

```diff
@@ -1,9 +1,9 @@
 .. _operations_overview :
 
-Operations
-==========
+Docker Operations
```

Comment:
do we have redirects getting setup for these renames?

Reply:
good reminder - i'll file in this PR https://github.com/confluentinc/docs/pull/605